AdminServer/appscale/admin/constants.py

```diff
@@ -0,0 +1,58 @@
+""" Constants used by AdminServer. """
+
+from appscale.common.constants import (
+  PYTHON27,
+  JAVA,
+  GO,
+  PHP
+)
+from tornado.web import HTTPError
+
+
+class CustomHTTPError(HTTPError):
+  """ An HTTPError that keeps track of keyword arguments. """
+  def __init__(self, status_code=500, **kwargs):
```

Comment:
`CustomHTTPError` is usually used with `message`argument. But `tornado.HTTPError` supports `reason` keyword argument. I think it makes sense to not limit basic functionality of `HTTPError` and pass all three argument which `HTTPError` handles: `status_code, log_message, reason`.